mods/cnc/rules/misc.yaml

```diff
@@ -67,4 +67,21 @@ CAMERA.small:
 	RevealsShroud:
 		Range: 6c0
 	BodyOrientation:
-
+	
+FLARE:
+	Immobile:
+		OccupiesSpace: false
+	Health:
```

Comment:
The flare doesn't need the Health trait. That's probably an error in the RA rules.